diff --git a/config/__pycache__/config.cpython-39.pyc b/config/__pycache__/config.cpython-39.pyc
index 31ec51c..81023fa 100644
Binary files a/config/__pycache__/config.cpython-39.pyc and b/config/__pycache__/config.cpython-39.pyc differ
diff --git a/scripts/train.py b/scripts/train.py
index c93aa82..440adeb 100644
--- a/scripts/train.py
+++ b/scripts/train.py
@@ -161,6 +161,5 @@ if __name__ == '__main__':
         wandb = None
 
     
-    # best_model_name = train_model(model, config, train_dataloader, valid_dataloader, wandb)
-    best_model_name = 'time=2022-11-17 06:57:42.718110_epoch=100.pt'
+    best_model_name = train_model(model, config, train_dataloader, valid_dataloader, wandb)
     validation_confusion_matrix(best_model_name, valid_dataloader, config, wandb)
